src/unix/evtloopunix.cpp

```diff
@@ -20,8 +20,6 @@
 // For compilers that support precompilation, includes "wx.h".
 #include "wx/wxprec.h"
 
-#if wxUSE_CONSOLE_EVENTLOOP
```

Comment:
Right, we don't make `wxUSE_EVENTLOOP_SOURCE` user-configurable currently because it's more of "whether we can have this feature or not" switch rather than "do we want to have it or not" (it should have been really called `wxHAVE_EVENTLOOP_SOURCE`).

Anyhow, looking at the source more carefully, I think what we really need is to add `wxUSE_CONSOLE_EVENTLOOP` check to the `#if` around `GetEventLoopSourcesManager()` in `include/wx/unix/apptbase.h` as this function really doesn't make any sense if we're not using the event loop in the first place. Could you please try fixing it like this this? TIA!